src/NLog.Web/README.md

```diff
@@ -0,0 +1,20 @@
+# NLog.Web
+
+- LoggerProvider for ASP.NET platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
+
+.NET 3.5 - 4.8
+
+Registration of NLog.Web in the NLog.config File
+	<!-- enable ASP.NET layout renderers -->
+	<extensions>
```

Comment:
Would it make sense to use code-markup as xml ?

Reply:
done